packages/fxa-settings/src/pages/ResetPassword/index.test.tsx

```diff
@@ -48,8 +64,7 @@ describe('PageResetPassword', () => {
     // when forceEmail is NOT provided as a prop, the optional read-only email should not be rendered
     const forcedEmailEl = screen.queryByTestId('reset-password-force-email');
     expect(forcedEmailEl).not.toBeInTheDocument();
-    // when 'canGoBack: false' or not passed as prop, the optional LinkRememberPassword component should not be rendered
-    expect(screen.queryByRole('link')).not.toBeInTheDocument();
+    screen.getByText('Remember your password? Sign in');
```

Comment:
We have been preferring accessible tests wherever possible - this could be `screen.getByRole('link',{name:'Remember your password? Sign in'})` to also check that this uses the right element.